dwave/cloud/config/models.py

```diff
@@ -99,6 +101,12 @@ class PollingSchedule(BaseModel, GetterMixin):
     backoff_base: Optional[float] = 1.3
 
 
+def _literal_eval(obj):
+    if isinstance(obj, str):
+        return ast.literal_eval(obj)
+    return obj
+
```

Comment:
Is this a leftover? I don't see it used (plus line 15)

Reply:
it's used in line 140 to define field validator

Reply:
Ah, there you go, thanks!